### DISCUSSION TITLE
Apollo Save Tool v1.4.2

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps4#145] [Announcements]
### Added

* Auto-detect `X`/`O` button settings (@illusion0001)
* Network HTTP proxy settings support
* New cheat codes
  - Grand Theft Auto V
  - Metal Gear Solid 5: The Phantom Pain
* Custom decryption support
  - Metal Gear Solid 5: The Phantom Pain
* Custom checksum support
  - Grand Theft Auto V
  - Metal Gear Solid 5: The Phantom Pain
  - Shantae and the Pirate's Curse
  - Shantae: Risky's Revenge

### Misc

* Include original keystones for 116 games
* Updated audio library to [`libs3m`](https://github.com/irqmask/s3mplay)
* Updated [`apollo-lib`](https://github.com/bucanero/apollo-lib) Patch Engine to v0.6.0
  - Add host callbacks (username, wlan mac, psid, account id)
  - Add `murmu3_32`, `jhash` hash functions
  - Add Patapon 3 PSP decryption
  - Add MGS5 decryption (PS3/PS4)
  - Add Monster Hunter 2G/3rd PSP decryption
  - Add Castlevania:LoS checksum
  - Add Rockstar checksum
  - Fix SaveWizard Code Type C
  - Fix `right()` on little-endian platforms

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps4/releases/tag/v1.4.2'>Apollo Save Tool v1.4.2</a>.</em>

**Comment by MDr4gonX** [comment]
Does the save work on Metal Gear Solid 5: The Definitive Experience?

**Comment by Lucatheoz** [comment]
If you could release a version of Apollo for Android it would be amazing

**Reply by Lucatheoz** [reply]
Maybe even carry it

**Comment by Lucatheoz** [comment]
If you could release a version of Apollo for Android it would be amazing
